vscode/package.json

```diff
@@ -156,6 +156,18 @@
           "command": "qsharp-vscode.getQir",
           "when": "resourceLangId == qsharp"
         },
+        {
+          "command": "qsharp-vscode.showHistogram",
+          "when": "resourceLangId == qsharp"
```

Comment:
Is it OK to show this command during a debug session? I'm guessing it won't interfere with the currently running program but the user also shouldn't expect that the histogram is relevant to the currently running program.

Reply:
Good catch. I could add that check and prohibit it. But as you outline, it runs in a separate worker, and there's actually no reason why you can't bring up a histogram for the code your debugging. Maybe that is interesting or useful when debugging something?

Reply:
Actually, I'll leave it. I see no reason to prohibit it.